File: Techschool/Techschool.BLL/Models/Vacations/AdditionalStudyVacationFormModel.cs
using System.Collections.Generic;

namespace Techschool.BLL.Models.Vacations
{
    public class AdditionalStudyVacationFormModel : VacationFormModel
    {
        public string AdditionalInfo { get; set; }
        public IEnumerable<AdditionalStudyVacationModel> AdditionalStudyVacations { get; set; }
    }
}

File: Techschool/Techschool/ClientApp/src/app/models/vacations/additional-study-vacation-form.model.ts
import { AdditionalStudyVacationModel } from './additional-study-vacation.model';
import { VacationFormModel } from './vacation-form.model';

export class AdditionalStudyVacationFormModel extends VacationFormModel {
  additionalInfo: string;
  additionalStudyVacations: AdditionalStudyVacationModel[];
}
